### PR TITLE
Feature/gal 602 error

### PR DESCRIPTION
**Pull request by Abdul-NHS**
https://nhsd-jira.digital.nhs.uk/browse/GAL-602

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by Amar-NHS — COMMENTED** [review]
just a little comment to address

File: src/app/views/clinic_information/ErrorBannerDetail.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      tabindex="-1"
+      onKeyDown={e => onKeyUp(e)}
+    >
+      <h2
+        class="nhsuk-error-summary__title nhsuk-u-margin-bottom-2"
+        id="error-target-title"
+      >
+        There is a problem
+      </h2>
+      <div class="nhsuk-error-summary__body">
```
[the remark this answers is not in the document]
Comment (NeerjaGoyalNHS) [reply src/app/views/clinic_information/ErrorBannerDetail.jsx:23]: changes done...

File: src/app/views/clinic_information/ClinicInformation.js [hunk 2]
```diff
@@ -53,7 +54,20 @@ class ClinicInformation extends Component {
   onClickGoBackLinkHandler() {
     this.context.setState({ "navigateToClinic": false })
     // Scroll to the top of the page every time it renders the page
-    // window.scrollTo(0, 0);
+    window.scrollTo(0, 0);
+  }
+
+  onKeyUp(e) {
+    console.log("enter is pressed", e);
```
[the remark this answers is not in the document]
Comment (NeerjaGoyalNHS) [reply src/app/views/clinic_information/ClinicInformation.js]: Done

**Review by Amar-NHS — DISMISSED** [review]
looks good. Approved.

**Review by Amar-NHS — APPROVED** [review]
approved

**Review by zainm97 — APPROVED** [review]

**Pull request merged**